include/nccl_ofi_rdma.h

```diff
@@ -758,6 +767,35 @@ struct nccl_net_ofi_rdma_ep {
 	/* thread id of the thread that called get_ep().  Used as the
 	   hash key for the endpoint hash */
 	long creating_thread_id;
+
```

Comment:
let's organize the code assuming that all of the fields above will eventually be private (ie, put the functions below with the other public functions above).